storage/netstore.go

```diff
@@ -176,6 +176,10 @@ func (n *NetStore) WithRecoveryCallback(f func(ctx context.Context, chunkAddress
 // If it is not found in the LocalStore then it uses RemoteGet to fetch from the network.
 func (n *NetStore) Get(ctx context.Context, mode chunk.ModeGet, req *Request) (ch Chunk, err error) {
 	metrics.GetOrRegisterCounter("netstore/get", nil).Inc(1)
+	publisher, ok := ctx.Value("publisher").(string)
+	if ok {
+		log.Debug("api.get", "netstore publisher", publisher)
+	}
```

Comment:
don't think this needs to be inside an `if` even if we keep the `log` call

Reply:
I would disagree, if we maintain this log and remove the `if` it would flood the console every time the Get is invoked, this should only show if there is a publisher.

The other option is to remove it, but I think this info could be useful for debugging purposes.

Reply:
i agree. so let's keep this, but please use a more descriptive string for the `Debug` call

Reply:
done thanks